payload/other/other.go

```diff
@@ -0,0 +1,31 @@
+// Other types of payloads that do not fit into the core categories.
+//
+// The payload package contains any specific payloads that do not fit into the other sub package types.
+package other
```

Comment:
I'd say this is the only thing I really don't like. other is pretty weak. fileplanting maybe? cron?